programs/mango-v4/src/instructions/account_buyback_fees_with_mngo.rs

```diff
@@ -38,14 +38,16 @@ pub fn account_buyback_fees_with_mngo(
     let now_ts = clock.unix_timestamp.try_into().unwrap();
     let slot = clock.slot;
 
+    let mngo_oracle_ref = &AccountInfoRef::borrow(&ctx.accounts.mngo_oracle.as_ref())?;
```

Comment:
aside: making these Refs like this can be dangerous because if the rust compiler doesn't drop() them asap, a future attempted mutable borrow could fail _at runtime_; it won't be an issue because no-one borrows oracles mutably, but it's something to be aware of. Previously the temporary values were guaranteed to be dropped at the end of the line.